lib/mayaUsd/CMakeLists.txt

```diff
@@ -267,7 +267,6 @@ target_link_libraries(${PROJECT_NAME}
         vt
         ${UFE_LIBRARY}
         ${MAYA_LIBRARIES}
-        mayaUsdUtils
```

Comment:
After moving diff/merge prims to UsdUfe this library was empty, so I removed it.